src/components/PoolOverview.tsx

```diff
@@ -58,6 +58,7 @@ export default function PoolOverview({
     sdlPerDay: formatBNToShortString(poolData?.sdlPerDay || Zero, 18),
   }
   const hasShare = !!userShareData?.usdBalance.gt(Zero)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Lost types or incompatible w\ latest version. Will have to prioritize router package upgrade.